simapp/test_helpers.go

```diff
@@ -312,11 +313,12 @@ func CheckBalance(t *testing.T, app *SimApp, addr sdk.AccAddress, balances sdk.C
 // the parameter 'expPass' against the result. A corresponding result is
 // returned.
 func SignCheckDeliver(
```

Comment:
we should prob unify the simapp/helpers/test_helpers.go and the simapp/test_helpers.go in a single file

Reply:
This will introduce a cyclic dependency, as simapp/test_helpers uses simapp for lot of it's functions. Not sure if it's doable

Reply:
We can remove simapp/helper and have only simapp/test_helpers though

Reply:
Moved simapp/helpers to simapp/test_helpers.go

Reply:
update: reverting this as it introduces cyclic dependency either way, and can't be resolved as package simulation needs helpers, module needs simulation and simapp imports module